Makefile

```diff
@@ -25,6 +25,8 @@ clean-pyc:
 lint: build
 	docker-compose run app flake8 --max-line-length 100 .
 	docker-compose run app yamllint repositories.yaml .circleci
+	docker-compose run app python -m black .
+	docker-compose run app python -m isort .
```

Comment:
Ditto, this will format sort imports instead of just checking.

Reply:
Yeah, that's true. Let me fix that up fast.

Reply:
@acmiyaguchi I thought of that but actually wondered if I would have to format the codebase as well before sending this PR.

All other PRs would start failing unless a formatted version of the code is merged.

Reply:
Yes, the whole code-base would need to be formatted and checked in.

@fbertsch what are your thoughts on formatting the codebase with `black`?

Reply:
Please do! Keep the reformatting as it's own separate commit.